integ-test/src/test/java/org/opensearch/sql/legacy/ExplainIT.java

```diff
@@ -271,4 +271,20 @@ public void testContentTypeOfExplainRequestShouldBeJson() throws IOException {
 
     assertEquals("application/json; charset=UTF-8", response.getHeader("content-type"));
   }
+
+  @Test
+  public void explainAlias() throws IOException {
+
+    String expectedOutputFilePath =
+        TestUtils.getResourceFilePath("src/test/resources/expectedOutput/alias_explain.json");
+    String expectedOutput =
+        Files.toString(new File(expectedOutputFilePath), StandardCharsets.UTF_8)
+            .replaceAll("\r", "");
+
+    String result =
+        explainQuery(
+            String.format("SELECT (age + 1) AS agePlusOne FROM %s LIMIT 10", TEST_INDEX_ACCOUNT));
```

Comment:
Could you overwrite some more complex cases:
1. SELECT \<field-list\> \<alias-list\>
2. SELECT field as alias1, alias1 as alias2, alias2 as alias3 (nested alias)
3. SELECT 1 as existing-field from t (override)

Reply:
1. Does SQL support such syntax? Or do you mean `filed1 as alias1, filed2 as alias2`?
2. Seems SQL don't support nested alias.
3. ditto.

Reply:
1. Compound case means not only contains pure fields but also contains fields with alias
2. It should be a bug if it's unsupported.
3. Same as above.

Reply:
1. I see. We have such cases in UT. Maybe make this IT be the compound case as well.
2. yeah. PPL supports while SQL doesn't.

Reply:
For 2 and 3, could you create two issues for that?

Reply:
Sure, but I think they may be the same cause. It's because we only use ProjectOperator instead of EvalOperator for SQL, while project don't support update and evaluate nested fields like eval.

Reply:
Create a new issue to bring out this problem here: https://github.com/opensearch-project/sql/issues/2933